libraries/AP_HAL_SITL/UARTDriver.cpp

```diff
@@ -160,7 +160,12 @@ size_t UARTDriver::write(const uint8_t *buffer, size_t size)
         DataFlash_Class::instance()->Log_Write("SBUS", "TimeUS,ToD", "QQ",
                                                AP_HAL::micros64(), time);
         // write buffer straight to the file descriptor
-        ::write(_fd, buffer, size);
+        ssize_t nwritten = ::write(_fd, buffer, size);
```

Comment:
`const` :-P

Reply:
yup, you're right (as usual)

Reply:
yep, guilty... and I am the one always saying to use it ...

Reply:
See PR https://github.com/ArduPilot/ardupilot/pull/7334